python/mxnet/contrib/quantization.py

```diff
@@ -499,6 +499,9 @@ def quantize_model(sym, arg_params, aux_params,
     if quantized_dtype not in ('int8', 'uint8'):
         raise ValueError('unknown quantized_dtype %s received,'
                          ' expected `int8` or `uint8`' % quantized_dtype)
+    if quantized_dtype == 'uint8' and ctx != cpu():
+        raise ValueError('currently gpu does not support uint8 quantization,'
+                         ' please set quantized_dtype to int8')
```

Comment:
How about the something like below？
“Currently, uint8 quantization is only supported by CPU, please switch to the context of CPU or int8 data type for GPU"

Reply:
okay, changed:)